### PR TITLE
Specify that you have non negotiable days off.

### PR DESCRIPTION
**Pull request by Eunoia**
Sometimes, people need to know that you go into the woods to meditate for 24 hours, and you take care of family.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +25,14 @@ Above script will create a git repository in the current directory. After that,
 
 `days` specify number of boxes you want to fill with green. Default value is 400. Remember, more commits, more green, more Rockstar you are!
 
-#### Become a Rockstar Swift Programmer with 5 years of experience:
+`days_off` tells the world that you're a rockstart who takes Wednesdays and Sundays to meditate, and work on your guitar solos.
```
Comment (avinassh) [inline README.md:28]: typo: `rockstart`

File: RockStar/__init__.py [hunk 2]
```diff
@@ -55,7 +56,10 @@ def _get_dates_list(self):
         dates_list = list()
         for day_delta in range(self.days):
             for i in range(randint(1, 10)):
-                dates_list.append(today - timedelta(days=day_delta))
+                day = today - timedelta(days=day_delta)
+                print day, day.strftime("%A"), day.strftime("%A") not in self.days_off
```
Comment (avinassh) [inline RockStar/__init__.py:60]: print is not required and this line will fail. Rockstar is Python 3 only! Please run and test the code before sending PR please.

**Comment by Eunoia** [comment]
Thanks for the very quick feedback. Looks like I've been using python2 :/

**Comment by avinassh** [comment]
Any update?

**Comment by avinassh** [comment]
#65 add this feature :)

**Pull request closed without merging**